pages/conf.py

```diff
@@ -24,18 +24,20 @@
 
 _extra_navbar = """
 <div class="sd-fs-6">
-<a  href="../en" target="_blank">en</a>
+<a href="../en">en</a>
 ·
-<a  href="../de" target="_blank">de</a>
+<a href="../de">de</a>
+·
+<a href="../zh_CN">zh_CN</a>
```

Comment:
Is `cn`, `zh` or the full `zh_CN` more intuitive here as language identifier?
```suggestion
<a href="../zh_CN">cn</a>
```

Reply:
`cn` seems fine, `zh_CN` name is indeed too long and eye-catching.